docs/01_getting_ready_for_class.md

```diff
@@ -77,7 +77,7 @@ If your clone is unsuccessful, read about [authenticating with GitHub from Git](
 
 #### Proxy configuration
 
-If your organization uses a proxy, you will need to configure the proxy settings in Git. Open Git Bash (on Windows) or Terminal (on Mac or *nix) and complete the appropriate steps below:
+If your organization uses a proxy, you will need to configure the proxy settings in Git. Open Git Bash (on Windows) or Terminal (on Mac or \*nix) and complete the appropriate steps below:
```

Comment:
May I ask why escaping the asterisk here is necessary? It seems to render fine in [the manual](https://githubtraining.github.io/training-manual/#/01_getting_ready_for_class?id=proxy-configuration) without the `\`.

Reply:
Just a quick test, the editor shows the line as bold without the `\` which might be why the escape character was included. The preview doesn't show the content as bold, but when writing in an editor it does 🤷 .

Reply:
I see. The `\` properly escapes the `*` in the GitHub editor and I verified that it doesn’t show up on the published web page by serving this branch locally, so this change looks good. 👍